spikeinterface/sortingcomponents/benchmark/benchmark_clustering.py

```diff
@@ -133,8 +144,9 @@ def run(self, peaks=None, positions=None, method=None, method_kwargs={}):
 
         self.matches = matches
         idx = matches['index1']
-        self.sliced_gt_sorting = NumpySorting.from_times_labels(times1[0][idx], times1[1][idx], self.sampling_rate)
-        
+        sorting_key = lambda x: int(''.join(filter(str.isdigit, x)))
```

Comment:
This is dangerous because some sorter have unit_ids int or str.
When str you are not always sure that it can be convertible to int. You could have "ch5#u12" for instance